docs/checker-setup.md

```diff
@@ -0,0 +1,60 @@
+# Setup checker
+
+The checker deals only with domains and TLS certifcate.
```

Comment:
This seems confusing to me. Doesn't the provider checker check all requirements?